downstream/titles/release-notes/async/aap-25-20250409.adoc

```diff
@@ -0,0 +1,100 @@
+[[aap-25-20250409]]
+
+= {PlatformNameShort} patch release April 9, 2025
+
+This release includes the following components and versions:
+
+[cols="1a,3a", options="header"]
+|===
+| Release date | Component versions
+
+| April 9, 2025  | 
+* {ControllerNameStart} 4.6.11
+* {HubNameStart} 4.10.3
+* {EDAName} 1.1.7
+* Container-based installer {PlatformNameShort} (bundle) 2.5-12
+* Container-based  installer {PlatformNameShort} (online) 2.5-12
+* Receptor 1.5.3
+* RPM-based installer {PlatformNameShort} (bundle) 2.5-11
+* RPM-based installer {PlatformNameShort} (online) 2.5-11
+
+|===
+
+CSV Versions in this release:
+
+* Namespace-scoped Bundle: aap-operator.v2.5.0-0.1743660124
+
+* Cluster-scoped Bundle: aap-operator.v2.5.0-0.1743660958
+
+== General
+
+* The `ansible.controller` collection has been updated to 4.6.11.(AAP-43126)
+
+* Fixed an issue where authentication configuration for *AzureAD/EntraId* groups could not be used in Authentication Mapping.(AAP-42890)
```

Comment:
Lower case "authentication mapping" as it's not a product name but a function.

Reply:
Done.